### ISSUE TITLE
Setup.exe error dialog 

### ISSUE THREAD

**Issue by grupobrik** [electron/grunt-electron-installer#62]
i run grunt create-windows-installer
after a couple of minutes it says Done, without errors 
but when i go to the output folder, there are the normal three files as expected, the RELEASES, .nupkg and .exe
when i execute the setup.exe file, it shows an error dialog saying:
This is a dummy update,exe, if you are seeing this, your build did not correctly replace the IDR_UPDATE_ZIP resource.
Command Line: '--install.'

i don´t know if this is really an issue or i missing something obvious, but i´m kind of noob at this and i don´t know where else ask for a little help 

this is my task config: 

'create-windows-installer': {
      x64: {
        appDirectory: '/Portafolio/PortafolioSinapsis-win32-x64',
        outputDirectory: '/build/installer64',
        authors: 'Sinapsis-innovation',
        remoteReleases: 'https://github.com/Sinapsis-Innovation/Portafolio-Sinapsis',
        exe: 'PortafolioSinapsis.exe'
      }
    },

Thanks for your help

**Comment by lookapanda** [comment]
Facing the same. x32 is created as expected, but the x64 setup.exe is only 200kb ....And shows the same error... :/ It worked several versions before :/

**Comment by paulcbetts** [comment]
When you run the Grunt job, in node_modules/grunt-electron-installer/vendor you should find a SquirrelSetup.log, what's in that file?

**Comment by havenchyk** [comment]
@grupobrik hi, is there any updates? Could you please attach log file?

**Comment by vasumahesh1** [comment]
Hi guys,

Facing the same issue for Both ia32 and x64 getting a 200-210Kb Setup for both cases...

`Failed to update Setup.exe with new Zip file`

Shouldn't the grunt task fail? I am getting a positive signal from the task ... 

x64
```
2015-12-26 15:28:11> Program: Starting Squirrel Updater: --releasify C:\Users\Vasu\AppData\Local\Temp\si1151126-20196-3bmw67\sampleapp.0.4.27.nupkg --releaseDir P:\forked-repos\testProject\dist\electron\windows\x64 --loadingGif P:\forked-repos\testProject\node_modules\grunt-electron-installer\resources\install-spinner.gif --no-msi
2015-12-26 15:28:11> Program: Bootstrapper EXE found at:P:\forked-repos\testProject\node_modules\grunt-electron-installer\vendor\Setup.exe
2015-12-26 15:28:11> Program: Creating release package: P:\forked-repos\testProject\dist\electron\windows\x64\sampleapp.0.4.27.nupkg
2015-12-26 15:28:11> ReleasePackage: Creating release package: P:\forked-repos\testProject\dist\electron\windows\x64\sampleapp.0.4.27.nupkg => P:\forked-repos\testProject\dist\electron\windows\x64\sampleapp-0.4.27-full.nupkg
2015-12-26 15:28:13> ReleasePackage: Extracting dependent packages: []
2015-12-26 15:28:13> ReleasePackage: Removing unnecessary data
2015-12-26 15:28:13> ReleasePackage: No release notes found in C:\Users\Vasu\AppData\Local\SquirrelTemp\tempa\sampleapp.nuspec
2015-12-26 15:28:20> Program: Building embedded zip file for Setup.exe
2015-12-26 15:28:22> Program: Failed to update Setup.exe with new Zip file: System.AggregateException: One or more errors occurred. ---> System.ComponentModel.Win32Exception: The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Squirrel.Utility.<InvokeProcessAsync>d__11.MoveNext()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at Squirrel.Update.Program.Releasify(String package, String targetDir, String packagesDir, String bootstrapperExe, String backgroundGif, String signingOpts, String baseUrl, String setupIcon, Boolean generateMsi)
---> (Inner Exception #0) System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Squirrel.Utility.<InvokeProcessAsync>d__11.MoveNext()<---
```

IA32:
```
2015-12-26 15:28:30> Program: Starting Squirrel Updater: --releasify C:\Users\Vasu\AppData\Local\Temp\si1151126-20196-u71x9\sampleapp.0.4.27.nupkg --releaseDir P:\forked-repos\testProject\dist\electron\windows\ia32 --loadingGif P:\forked-repos\testProject\node_modules\grunt-electron-installer\resources\install-spinner.gif --no-msi
2015-12-26 15:28:30> Program: Bootstrapper EXE found at:P:\forked-repos\testProject\node_modules\grunt-electron-installer\vendor\Setup.exe
2015-12-26 15:28:30> Program: Creating release package: P:\forked-repos\testProject\dist\electron\windows\ia32\sampleapp.0.4.27.nupkg
2015-12-26 15:28:30> ReleasePackage: Creating release package: P:\forked-repos\testProject\dist\electron\windows\ia32\sampleapp.0.4.27.nupkg => P:\forked-repos\testProject\dist\electron\windows\ia32\sampleapp-0.4.27-full.nupkg
2015-12-26 15:28:32> ReleasePackage: Extracting dependent packages: []
2015-12-26 15:28:32> ReleasePackage: Removing unnecessary data
2015-12-26 15:28:32> ReleasePackage: No release notes found in C:\Users\Vasu\AppData\Local\SquirrelTemp\tempa\sampleapp.nuspec
2015-12-26 15:28:37> Program: Building embedded zip file for Setup.exe
2015-12-26 15:28:39> Program: Failed to update Setup.exe with new Zip file: System.AggregateException: One or more errors occurred. ---> System.ComponentModel.Win32Exception: The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Squirrel.Utility.<InvokeProcessAsync>d__11.MoveNext()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at Squirrel.Update.Program.Releasify(String package, String targetDir, String packagesDir, String bootstrapperExe, String backgroundGif, String signingOpts, String baseUrl, String setupIcon, Boolean generateMsi)
---> (Inner Exception #0) System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Squirrel.Utility.<InvokeProcessAsync>d__11.MoveNext()<---
```

**Comment by shoaibmerchant** [comment]
Hi any ideas on this issue?

**Comment by paulcbetts** [comment]
This is fixed in #97

**Comment by dominicrico** [comment]
Getting this error with the latest version and Windows 10 when creating a x64 version of my app.
